app/cronjob/trailingTradeHelper/configuration.js

```diff
@@ -202,6 +202,74 @@ const getMaxPurchaseAmount = async (
 
   return newBuyMaxPurchaseAmount;
 };
+
+const getLastBuyThreshold = async (
+  logger,
+  symbol,
+  globalConfiguration,
+  symbolConfiguration
+) => {
+  const symbolBuyLastBuyThreshold = _.get(
+    symbolConfiguration,
+    'buy.lastBuyThreshold',
+    10
```

Comment:
`10` doesn't make sense. 

The reason `getMaxPurchaseAmount` having `-1` as default value is because to determine whether the value is set or not. `10` is already a correct threadhold value. 

In the next `if` statement, you check 10, which is not making sense. Does it mean `10` threshold is ignored?

Reply:
When you said it, it made sense. Fixing...